r/vignettes/developers/docker.Rmd

```diff
@@ -0,0 +1,71 @@
+# Using docker with Arrow
+
+Arrow is compatible with a huge number of combinations of OSs, OS versions, 
+compilers, R versions, and other variables.  Sometimes these combinations of 
```

Comment:
> a huge number of combinations of OSs, OS versions, compilers, R versions, and other variable

I feel like this is one of those lines that might be prefaced with "~unfortunately~ fortunately, ... " 😂 (not actually suggesting to add this, just commenting on the vibe)